calibration:
  ros__parameters:
    # 世界坐标点配置
    world_points:
      self_fortress: [5.471, -7.5, 0.0]
      self_tower: [10.936, -11.161, 0.868]
      enemy_base: [25.49, -7.5, 1.24524]
      enemy_tower: [16.925, -3.625, 1.745]
      enemy_high: [20.20, -10.8, 0.8]
    
    # 窗口配置
    window:
      width: 1440
      height: 900
      pos_x: 490 #1920-1440
      pos_y: 180 #1080-900
      roi_weight: 400
      roi_height: 400
    
    # 图像处理配置
    image:
      width: 1536
      height: 1125
      scale_factor: 1.3333333333
      roi_zoom_size: 100
    
    # 标定参数
    calibration:
      pnp_method: "EPNP"
      camera_params_path: "/home/chichu/Lidar/ITL_ws/src/itl_vision/config/camera_params.yaml"
      output_path: "src/itl_vision/config/out_matrix.yaml"

    # 话题配置
    topics:
      image_topic: "image_raw"
      compressed_topic: "compressed_image"